.rubocop.yml

```diff
@@ -1,12 +1,18 @@
 inherit_from:
   - .rubocop_rspec_base.yml
+  - .rubocop_todo.yml
 
+
+
+# Over time we'd like to get this down, but this is what we're at now.
 Metrics/AbcSize:
   Max: 28
 
+# Over time we'd like to get this down, but this is what we're at now.
 Metrics/BlockLength:
-  Max: 86
+  Max: 834
```

Comment:
This is a very large increase for support, is this file just copied and pasted?

Reply:
No, it wasn't.  It was necessary to accomodate the blocks in `spec`.  Excluding those allows us to go back to the original value.  I've update the PR.